### PR TITLE
Added contact form on the mobile portfolio

### PR DESCRIPTION
**Pull request by yayner2002**
Here we have added the contact form for the mobile version portfolio website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -226,5 +227,46 @@ <h3>
...
+          <li>
+            <a href="#"><i class="fa-brands fa-angellist"></i></a>
+          </li>
+        </ul>
+      </section>
+      <section class="email">
+        <a href="mailto:yayner2002@gmail.com">Yayner2002@gmail.com</a>
+      </section>
+      <img src="images/Shape.svg" alt="contact-shape" class="contact-shape">
+  </footer>
```
Comment (ShadyShawkat) [inline index.html]: Kindly try to stick as much as possible to the Figma design, which includes:
   - [x] Adjusting the background image position to match the Figma design.
   - [x] Adding the button interactions (hover, pressed, ...etc)

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @yayner2002, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (yayner2002) [reply index.html]: Here we have managed to update the background image position and the hover effect of the submit button as per your request.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @yayner2002, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**